internal/metamorph/store/store.go

```diff
@@ -12,33 +12,33 @@ import (
 
 var ErrNotFound = errors.New("key could not be found")
 
-type StoreData struct {
+type Data struct {
 	RawTx             []byte
 	StoredAt          time.Time
 	LastModified      time.Time
 	Hash              *chainhash.Hash
 	Status            metamorph_api.Status
 	BlockHeight       uint64
 	BlockHash         *chainhash.Hash
-	Callbacks         []StoreCallback
-	StatusHistory     []*StoreStatus
+	Callbacks         []Callback
+	StatusHistory     []*Status
 	FullStatusUpdates bool
 	RejectReason      string
 	CompetingTxs      []string
 	LockedBy          string
-	Ttl               int64
+	TTL               int64
 	MerklePath        string
 	LastSubmittedAt   time.Time
 	Retries           int
 }
 
-type StoreCallback struct {
+type Callback struct {
 	CallbackURL   string `json:"callback_url"`
 	CallbackToken string `json:"callback_token"`
 	AllowBatch    bool   `json:"allow_batch"`
 }
 
-type StoreStatus struct {
+type Status struct {
```

Comment:
Minor
@arkadiuszos4chain will probably like this, but we should consciously decide if we want these changes such as `StoreCallback` to just `Callback`.

Reply:
IMO the change is good. As the package is called `store`, the `Store` in `StoreCallback` is kind of redundant